setup.cfg

```diff
@@ -20,6 +20,10 @@ known_third_party = brotli,certifi,chardet,cryptography,h11,h2,hstspreload,pytes
 skip = httpcore/_sync/,tests/sync_tests/
 
 [tool:pytest]
-addopts = --cov-report= --cov=httpcore --cov=tests -rxXs
+addopts = -rxXs
```

Comment:
I'd love for us to be using long form flags here at some point. I mean, what the heck does *this do*?

(Tho, that's nothing to do with this pull request, just makes me notice it more is all.)

Reply:
Magic incantation ✨

Reply:
Let's make the flags wide (wait a second please)

Reply:
afaik you cant make reporting flags wide, 
```
  -r chars              show extra test summary info as specified by chars: (f)ailed, (E)rror, (s)kipped, (x)failed, (X)passed, (p)assed, (P)assed with output, (a)ll except passed (p/P), or (A)ll. (w)arnings are enabled by default (see --disable-warnings), 'N' can
                        be used to reset the list. (default: 'fE').
```

Reply:
Ha :-) There is no long option for `-r`. The flag combination means:
`-r`: show extra test summary info as specified by chars: (s)kipped, (x)failed, (X)passed

Reply:
Well, we oughta treat it separately in any case. Perhaps we don't really want it at all.